libs/membership/api/src/lib/dashboard/invoices/dashboard-invoice.service.ts

```diff
@@ -0,0 +1,102 @@
+import {Injectable} from '@nestjs/common'
+import {PrismaClient} from '@prisma/client'
+import {DashboardInvoice} from './dashboard-invoice.model'
+import {ok} from 'assert'
+
+@Injectable()
+export class DashboardInvoiceService {
```

Comment:
Very much like, that we use models and services along with classes. But same problem here: Can we simplify the path?